readme.md

```diff
@@ -12,31 +12,74 @@
 
 ## Framing (5 minutes / 0:05)
 
-In order to do things on the client side and give our web applications behavior, we need programmatic access to the HTML and CSS using Javascript. Javascript gives us not only the ability to manipulate the **DOM**, but also to make it respond to user actions. This is where events come in: we can *listen* for certain kinds of user-driven events, such as clicking a button, entering data into a form, keypresses and many, many more.
+So far, we have needed to use the REPL in the browser console to interact with our programs.
+This is asking a bit much of our users.
+Instead we would like to write code to respond to user interactions with the webpage.
+The **DOM** not only lets us manipulate the document or webpage using JavaScript, but also gives us the ability to write JavaScript that responds to interactions with the page.
+These interactions are communicated as **events**.
+
+> What might be an event we would want to respond to with JavaScript?
+
+We can *listen* for certain kinds of user-driven events, such as clicking a button, entering data into a form, keypresses and many, many more.
+
+## You Do: What Is An Event? (10 minutes / 0:10)
+
+> 7 minutes exercise. 3 minutes review.
+
+Spend 7 minutes doing the following tasks. You are encouraged to discuss your findings with a partner during the exercise.
+
+1. Come up with your own definition without looking at any other sources. Don't worry about getting it right -- what do you think an event is?
+2. Now, find (i.e., Google) some documentation on Javascript events. Does that information match your definition? How would you update your definition to include what you have learned but rephrased in your own words?
+3. Write down three examples of events.
+
+> If you need some help, you can find information on events and examples [W3Schools](http://www.w3schools.com/js/js_events.asp) and [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Events).
+
+When you have finished, read [this issue](https://github.com/ga-wdi-lessons/js-events-callbacks/issues/9) and submit a comment with the three parts clearly labeled.
```

Comment:
How are the responses on the issue going to be used? Are you going to go through them on screen?